certified-connectors/Knowledgeone RecFind 6/apiProperties.json

```diff
@@ -0,0 +1,31 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "api_key": {
+        "type": "securestring",
+        "uiDefinition": {
+          "displayName": "Security Key",
+          "description": "The Security Key for this api",
+          "tooltip": "Provide your Security Key",
+          "constraints": {
+            "tabIndex": 2,
+            "clearText": false,
+            "required": "true"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#FB8003",
+    "capabilities": [],
+    "policyTemplateInstances": [
+      {
+        "templateId": "dynamichosturl",
+        "title": "HostUrl",
+        "parameters": {
+          "x-ms-apimTemplateParameter.urlTemplate": "@queryParameters('HostUrl')"
+        }
+      }
+    ],
+    "publisher": "Emmanuel Cardakaris"
```

Comment:
@ecardakaris-knowledgeonecorp Please go thru the instructions mentioned in the [documentation](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#step-3-add-metadata) to add metadata for your connector. 

An individual or personal name is not allowed for "publisher" property as your connector will be as deployed as a certified-connector. Please provide your company or organization name. And add a stackOwner property.